testing/src/main/java/io/stargate/it/cql/AbstractCompressionTest.java

```diff
@@ -40,6 +40,6 @@ protected void compressionTest(CqlSession session) {
     // We are testing with small responses, so the compressed payload is not even guaranteed to be
     // smaller.
     assertThat(executionInfo.getResponseSizeInBytes()).isGreaterThan(0);
-    assertThat(executionInfo.getCompressedResponseSizeInBytes()).isGreaterThan(0);
+    // assertThat(executionInfo.getCompressedResponseSizeInBytes()).isGreaterThan(0);
```

Comment:
Driver is always returning the value of executionInfo.getCompressedResponseSizeInBytes() as -1. Behavior is same even when ran against cassandra 4.0.3 directly.

Reply:
Does `-1` signify "Not known" (or "Not available") -- and if so, is that a legal response?
Or does this indicate that compression is not being used?

Reply:
-1 is the default value set for compressed size. I ran test in debug, it goes through compression.

Reply:
Ok. What I was wondering was whether it is legal to return `-1` when compression is used, or only if no compression is used. If former, it would still be compliant implementation and test could pass. That does not mean we have to accept it in the test, maybe code is incorrectly omitting setting the size somewhere and we really should non-zero positive value.